app/views/expenses/new.html.erb

```diff
@@ -0,0 +1,28 @@
+<%# path = app/views/expenses/new.html.erb %>
+<div class="categories-index">
+  <div class="header">
+    <span><a href="<%= category_path %>" class="back-arrow">←</a></span>
```

Comment:
- [ ] Kindly fix this error
![image](https://github.com/noelfoka/capstone_budget_app/assets/24703171/24b0f40a-c793-42d6-b36c-9b59a6413834)

I get this error when I visit the "Add a new transaction" page, so I wasn't able to check the problem with the Back button as requested by the previous reviewer.